s3-import-commons/src/main/java/no/unit/nva/publication/s3imports/FilenameEventEmitter.java

```diff
@@ -114,12 +122,18 @@ private void logWarningForNotEmittedFilenames(List<PutEventsResult> failedReques
                                eventBridgeClient);
 
         List<ImportRequest> filenameEvents = files.stream()
-                                                 .map(uri -> new ImportRequest(uri, publicationsOwner))
+                                                 .map(uri -> newImportRequestForSingleFile(importRequest, uri))
                                                  .collect(Collectors.toList());
         eventEmitter.addEvents(filenameEvents);
```

Comment:
A FilenameEvent is an ImportRequest?

Reply:
yes. The class FilenameEvent does not exist any longer. One can have an import request for a folder or for a file